prime-router/src/main/kotlin/Translator.kt

```diff
@@ -239,15 +240,16 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
     fun translateByReceiver(
         input: Report,
         receiver: Receiver,
-        defaultValues: DefaultValues = emptyMap()
+        defaultValues: DefaultValues = emptyMap(),
+        generatingEmpty: Boolean = false
```

Comment:
Doesn't look like this is used anywhere, it seems to be always set to false.

Reply:
Artifact, removed.